.phpunit.result.cache

```diff
@@ -0,0 +1 @@
+C:37:"PHPUnit\Runner\DefaultTestResultCache":2085:{a:2:{s:7:"defects";a:0:{}s:5:"times";a:25:{s:51:"Commercie\Tests\Currency\CurrencyTest::testGetLabel";d:0.001;s:58:"Commercie\Tests\Currency\CurrencyTest::testGetRoundingStep";d:0;s:68:"Commercie\Tests\Currency\CurrencyTest::testGetRoundingStepBySubunits";d:0;s:69:"Commercie\Tests\Currency\CurrencyTest::testGetRoundingStepUnavailable";d:0;s:54:"Commercie\Tests\Currency\CurrencyTest::testGetDecimals";d:0;s:53:"Commercie\Tests\Currency\CurrencyTest::testIsObsolete";d:0;s:62:"Commercie\Tests\Currency\CurrencyTest::testGetAlternativeSigns";d:0;s:58:"Commercie\Tests\Currency\CurrencyTest::testGetCurrencyCode";d:0;s:60:"Commercie\Tests\Currency\CurrencyTest::testGetCurrencyNumber";d:0;s:50:"Commercie\Tests\Currency\CurrencyTest::testGetSign";d:0;s:54:"Commercie\Tests\Currency\CurrencyTest::testGetSubunits";d:0;s:51:"Commercie\Tests\Currency\CurrencyTest::testGetUsage";d:0;s:51:"Commercie\Tests\Currency\InputTest::testParseAmount";d:0;s:67:"Commercie\Tests\Currency\ResourceRepositoryTest::testListCurrencies";d:0.001;s:65:"Commercie\Tests\Currency\ResourceRepositoryTest::testLoadCurrency";d:0.004;s:107:"Commercie\Tests\Currency\ResourceRepositoryTest::testLoadCurrencyWithNonExistentCurrencies with data set #0";d:0;s:107:"Commercie\Tests\Currency\ResourceRepositoryTest::testLoadCurrencyWithNonExistentCurrencies with data set #1";d:0;s:107:"Commercie\Tests\Currency\ResourceRepositoryTest::testLoadCurrencyWithNonExistentCurrencies with data set #2";d:0;s:107:"Commercie\Tests\Currency\ResourceRepositoryTest::testLoadCurrencyWithNonExistentCurrencies with data set #3";d:0;s:107:"Commercie\Tests\Currency\ResourceRepositoryTest::testLoadCurrencyWithNonExistentCurrencies with data set #4";d:0;s:107:"Commercie\Tests\Currency\ResourceRepositoryTest::testLoadCurrencyWithNonExistentCurrencies with data set #5";d:0;s:57:"Commercie\Tests\Currency\Resources::testCurrencyIntegrity";d:0.029;s:48:"Commercie\Tests\Currency\UsageTest::testGetStart";d:0;s:46:"Commercie\Tests\Currency\UsageTest::testGetEnd";d:0;s:54:"Commercie\Tests\Currency\UsageTest::testGetCountryCode";d:0;}}}
```

Comment:
This file must be removed. It was probably added before `.gitignore` was updated :)